src/main/java/se/munhunger/idp/dao/UserDAO.java

```diff
@@ -26,4 +29,23 @@ public void createUser(User user) {
             session.getTransaction().commit();
         }
     }
+
+    public void updateUser(User user) throws NotInDatabaseException {
+        try(Session session = sessionFactory.openSession()) {
+            Optional<User> tempUser = getUser(user.getUsername());
+            tempUser.orElseThrow(NotInDatabaseException::new);
+            session.merge(tempUser.get());
```

Comment:
Varför inte köra orElseThrow direkt här?

Reply:
Testade: session.merge(tempUser.orElseThrow(NotInDatabaseException::new));
Och det fungerade, jag trodde inte att merge skulle kunna hantera Optional<User> utan bara kan hantera en User

Plus så fastnade jag lite med pga bygget inte ville riktigt då jag fick java.lang.NoClassDefFoundError efter att ha ändrat Util till util package fick gå in och ändra det manuellt